app/uk/gov/hmrc/minorentityidentificationfrontend/services/RegistrationOrchestrationService.scala

```diff
@@ -28,14 +29,23 @@ class RegistrationOrchestrationService @Inject()(storageService: StorageService,
                                                  registrationConnector: RegistrationConnector,
                                                  auditService: AuditService) {
 
+  private def internalRegister(utr: String, journeyConfig: JourneyConfig)
+                      (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[RegistrationStatus] = {
+    journeyConfig.businessEntity match {
+      case Trusts => registrationConnector.registerTrust(utr, journeyConfig.regime)
+      case UnincorporatedAssociation => registrationConnector.registerUA(utr, journeyConfig.regime)
+      case OverseasCompany => throw new IllegalArgumentException("Overseas Company is not supported for registration")
+    }
+  }
+
   def register(journeyId: String,
-               optSautr: Option[String],
+               optUtr: Option[String],
                journeyConfig: JourneyConfig
-              )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[RegistrationStatus] = for {
+                   )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[RegistrationStatus] = for {
     registrationStatus <- storageService.retrieveBusinessVerificationStatus(journeyId).flatMap {
-      case Some(BusinessVerificationPass) => registrationConnector.register(optSautr.get, journeyConfig.regime)
-      case None if !journeyConfig.businessVerificationCheck && optSautr.isDefined =>
-        registrationConnector.register(optSautr.get, journeyConfig.regime)
+      case Some(BusinessVerificationPass) => internalRegister(optUtr.get, journeyConfig)
```

Comment:
can we format this  some are following the → one line is below the case None .... let's just use one approach. Whatever dont mind